src/data/skill-desc.csv

```diff
@@ -384,6 +385,7 @@
 "最終コーナーで前の方にいると\n速度がわずかに上がる","<size=22>Slightly increase speed if you're at the front after the last corner</size>"
 "最終直線で余力を持って前にいると\nちょっと優位を保ちやすくなる","<size=22>If you're in front during the last straight with spare energy, it becomes easier to keep a good position</size>"
 "最終コーナーで前の方で競り合うと\n速度がわずかに上がる","<size=22>Fight for the lead at the last corner to slightly increase your speed</size>"
+"最終コーナー以降で外から追い抜くと\n速度をわずかに上げる",""<size=22>Slightly increase speed on the final corner when passing from the outside</size>""
```

Comment:
"From the last corner onward" would be more precise

Reply:
i tried keeping it short, also keeping consistency with other same 最終コーナー以降で- skill

Reply:
imo this is a pretty significant difference, since you don't only speed up in the last corner -- the other skill should then be adjusted as well, i'd say

Reply:
alright, will try to adjust the other one as well then

Reply:
you can cut the `the` if length seems concerning